plugin/vcs/gitlab/gitlab.go

```diff
@@ -298,10 +298,15 @@ func (provider *Provider) FetchRepositoryActiveMemberList(ctx context.Context, o
 			if err != nil {
 				return nil, err
 			}
+
+			if userInfo.PublicEmail == "" {
```

Comment:
Is PublicEmail optional from gitlab side?

Reply:
Users can choose not to show this field.

Reply:
Then are we failing the whole sync process if one is missing?

Reply:
The current logic is your case. Or maybe we can just pass this user?

Reply:
But by default, GitLab will set public email to null, unless users manually config theirs.